src/core/server/saved_objects/routes/delete.ts

```diff
@@ -32,11 +32,13 @@ export const registerDeleteRoute = (router: IRouter, { coreUsageData }: RouteDep
     catchAndReturnBoomErrors(async (context, req, res) => {
       const { type, id } = req.params;
       const { force } = req.query;
+      const { getClient } = context.core.savedObjects;
 
       const usageStatsClient = coreUsageData.getClient();
       usageStatsClient.incrementSavedObjectsDelete({ request: req }).catch(() => {});
 
-      const result = await context.core.savedObjects.client.delete(type, id, { force });
+      const client = getClient();
```

Comment:
No changes to the implementation here; just changed from `.client` to `.getClient` to stay consistent with the other routes.